wagtail/utils/sendfile_streaming_backend.py

```diff
@@ -19,7 +19,9 @@ def sendfile(request, filename, **kwargs):
     ):
         return HttpResponseNotModified()
 
-    response = FileResponse(open(filename, "rb"))
+    response = {}
+    with open(filename, "rb") as f:
+        response = FileResponse(f)
```

Comment:
Issue: This change isn't necessary, and probably won't work. If you pass an open file descriptor to a response, Django automatically closes it when it's finished sending its contents.

> The file will be closed automatically, so don’t open it with a context manager.

https://docs.djangoproject.com/en/5.1/ref/request-response/#fileresponse-objects